src/domain/session/room/LightboxViewModel.js

```diff
@@ -19,21 +19,25 @@ import {ViewModel} from "../../ViewModel";
 export class LightboxViewModel extends ViewModel {
     constructor(options) {
         super(options);
-        this._eventId = options.eventId;
+        this._eventEntry = options.eventEntry;
+        this._eventId = options.eventId || options.eventEntry.id;
         this._unencryptedImageUrl = null;
         this._decryptedImage = null;
         this._closeUrl = this.urlCreator.urlUntilSegment("room");
-        this._eventEntry = null;
         this._date = null;
         this._subscribeToEvent(options.room, options.eventId);
     }
 
     _subscribeToEvent(room, eventId) {
-        const eventObservable = room.observeEvent(eventId);
-        this.track(eventObservable.subscribe(eventEntry => {
-            this._loadEvent(room, eventEntry);
-        }));
-        this._loadEvent(room, eventObservable.get());
+        let event = this._eventEntry;
+        if (!this._eventEntry) {
```

Comment:
Added the option of passing in `eventEntry` directly since we have access to all of the events already in [`matrix-public-archive`](https://github.com/matrix-org/matrix-public-archive/pull/12) and we don't need or want to load more events.

Previously, this only allowed passing `eventId` which loaded in the full event.